.github/workflows/prover-daily-test.yaml

```diff
@@ -38,5 +38,7 @@ jobs:
           echo "/home/runner/bin" | tee -a $GITHUB_PATH
           echo "/home/runner/.dotnet" | tee -a $GITHUB_PATH
           echo "/home/runner/.dotnet/tools" | tee -a $GITHUB_PATH
-      - run: MVP_TEST_INCONSISTENCY=1 cargo test -p aptos-move-examples --release -- --include-ignored prover
-      - run: MVP_TEST_INCONSISTENCY=1 cargo test -p aptos-framework --release -- --include-ignored prover
+      - run: MVP_TEST_DISALLOW_TIMEOUT_OVERWRITE=1 MVP_TEST_VC_TIMEOUT=1200 cargo test -p aptos-move-examples --release -- --include-ignored prover
```

Comment:
Does this `--disable-global-timeout-overwrite` force Prover to the global timeout limit, ignoring the local ones?

Reply:
yes